### PR TITLE
Dev change

### PR DESCRIPTION
**Pull request by ajphukan**
Formatting is done .
Anything else i forgot do .

### CODE REVIEW

**Comment by andy5995** [comment]
Looks good. Thank you. :)

**Comment by ajphukan** [comment]
@andy5995 why we are calling getenv() more than two times  ?

File: src/main.c [hunk 1]
```diff
@@ -128,9 +128,17 @@ main (int argc, char *argv[])
...
   while (next_option != -1);
 
-  bufchk (getenv ("HOME"), MP);
+  if (getenv ("HOME") == NULL)
+  {
+    fprintf (stderr, "Error: Environmental variable $HOME can't be used. Unable to determine home directory\n");
+    return 1;
+  }
+  
   char HOMEDIR[strlen (getenv ("HOME")) + 1];
```
Comment (andy5995) [inline src/main.c]: When the length of a string is known, I prefer to use it when declaring the size of an array. Although I'm using getenv() more than once, that's ok.

Comment (ajphukan) [inline src/main.c]: but when getenv is called it already reserved some space in memory ..

check if you like the change or else it is ok : 
```c
 char *HOMEDIR = getenv ("HOME");
 if (HOMEDIR == NULL)
  {
    fprintf (stderr, "Error: Environmental variable $HOME can't be used. Unable to determine home directory\n");
    return 1;
  }
 
  if (bufchk (HOMEDIR, MP))
  return BUF_ERR;
```

Comment (andy5995) [inline src/main.c]: I think I understand what you're saying. Go ahead and make the change.

**Pull request merged**